### DISCUSSION TITLE
gdscript editor adds blank lines

### DISCUSSION THREAD

**Discussion by lionet1224** [godotengine/godot-proposals#8421] [Editor]
### Describe the project you are working on

During development with godot, when more code is developed in gdscript's editor, the code will be at the bottom of the editor when the code is larger than one page.

### Describe the problem or limitation you are having in your project

In most major code editors, to optimize the development experience, some blank lines are left at the bottom so that the code doesn't hit the bottom.

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Add blank lines

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

-

### If this enhancement will not be used often, can it be worked around with a few lines of script?

-

### Is there a reason why this should be core and not an add-on in the asset library?

-

**Comment by lionet1224** [comment]
phpstorm
<img width="1032" alt="image" src="https://github.com/godotengine/godot-proposals/assets/35787877/00721b78-4fe2-4c29-9d33-4b13affaea34">
vscode
<img width="1203" alt="image" src="https://github.com/godotengine/godot-proposals/assets/35787877/2929284c-beac-4819-b89f-1ab475e6b9b7">

**Comment by AThousandShips** [comment]
Converted to discussion as no details on how to achieve this were given

**Comment by Calinou** [comment]
You can enable **Scroll Past End** in the Editor Settings to achieve this. VS Code also has this option – it just happens to be enabled by default.

**Reply by lionet1224** [reply]
Okay, thanks.
